students/daniel_carrasco/session09/donor_database.py

```diff
@@ -0,0 +1,47 @@
+class Donor_Database:
+    data = {}
+
+    def __init__(self, name=""):
```

Comment:
I would think you would always want `name` to be required for the instantiation of the class.